featurebyte/query_graph/sql/ast/tile.py

```diff
@@ -36,7 +36,10 @@ class BuildTileNode(TableNode):
     value_by: str | None
     tile_specs: list[TileSpec]
     timestamp: str
-    frequency: int
+    frequency_minute: int
+    blind_spot: int
+    time_modulo_frequency: int
```

Comment:
What does `frequency_minute` mean? Would be good to leave some comments on the input to `F_TIMESTAMP_TO_INDEX` here.

Reply:
This is the frequency component of the feature job settings but using minute as the unit instead of second. This is a requirement of the `F_TIMESTAMP_TO_INDEX` function. Will add comments in a next PR.